Expand Difficulty Increase mod articles (#12629)
ppy/osu-wiki: wiki/Gameplay/Game_modifier/Strict_Tracking/en.md

*For the full list of all [lazer](/wiki/Client/Release_stream/Lazer) mods, see: [Game modifier (lazer)](/wiki/Gameplay/Game_modifier_(lazer))*

> The **Strict Tracking** mod causes a miss every time you aren't precisely following the [slider](/wiki/Gameplay/Hit_object/Slider) follow circle. The reasons for the miss can be missing a [slider tick](/wiki/Gameplay/Hit_object/Slider/Slider_tick), a [slidertail](/wiki/Gameplay/Hit_object/Slider/Slidertail), not holding through an empty slider path and being outside of the follow circle regardless of duration.

Reviewer: ```suggestion
The **Strict Tracking** mod causes a miss every time the player isn't precisely following the [slider](/wiki/Gameplay/Hit_object/Slider) follow circle. This includes missing a [slider tick](/wiki/Gameplay/Hit_object/Slider/Slider_tick), a [slidertail](/wiki/Gameplay/Hit_object/Slider/Slidertail), not holding through an empty slider path, and being outside of the follow circle for any amount of time.
```
